string input
---
int8 num_objects
ReconfigurationObject[] objects
string validMove